sources/Outputs/include/Job.h

```diff
@@ -89,9 +90,14 @@ class Job {
    */
   boost::shared_ptr<job::JobEntry> write();
 
+  static void setStartAndDuration(uint32_t startTime, uint32_t durationTime) {
```

Comment:
Accept parameters as `double` or `float`, whatever we decide. We have to rethink on having these attributes as `static`.